.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+    "env": {
+      "browser": true,
+      "jest": true,
+      "node": true
+    },
+    "settings": {
+      "react": {
+        "pragma": "React",
+        "version": "^16.4.2"
+      }
+    },
+    "parser": "babel-eslint",
+    "plugins": [
+      "react"
+    ],
+    "rules": {
```

Comment:
Why not using the `airbnb` config?

Reply:
We are using airbnb. Please check package.json

Reply:
Moved extension part to eslintrc